SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
Hmm, then maybe set a high minLevel like ERROR and then all filters are used? Would that work?